### PR TITLE
Add script sys path back to test

### PR DESCRIPTION
**Pull request by aspfohl**

### CODE REVIEW

File: tests/a_scripts/inference/test_convert_composer_to_hf.py [hunk 1]
```diff
@@ -4,26 +4,34 @@
 import math
 import os
 import pathlib
+import sys
+from typing import Callable
```
Comment (vchiley) [inline tests/a_scripts/inference/test_convert_composer_to_hf.py:8]:
```suggestion
from typing import Callable, Optional, cast
```

File: tests/a_scripts/inference/test_convert_composer_to_hf.py [hunk 2]
```diff
@@ -4,26 +4,34 @@
...
+from llmfoundry.models.mpt.modeling_mpt import ComposerMPTCausalLM
+
+# Add repo root to path so we can import scripts and test it
+repo_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))
+sys.path.append(repo_dir)
 import shutil
 from argparse import Namespace
-from typing import Callable, Optional, cast
-from unittest.mock import ANY, MagicMock, patch
+from typing import Optional, cast
```
Comment (vchiley) [inline tests/a_scripts/inference/test_convert_composer_to_hf.py:23]:
```suggestion
```

**Review by vchiley — APPROVED** [review]

File: tests/a_scripts/inference/test_convert_composer_to_hf.py [hunk 3]
```diff
@@ -4,26 +4,34 @@
 import math
 import os
 import pathlib
+import sys
+from typing import Callable
+from unittest.mock import ANY, MagicMock, patch
+
+from composer import Trainer
+from composer.loggers import MLFlowLogger
+from composer.utils import dist, get_device, using_torch_2
```
Comment (vchiley) [inline tests/a_scripts/inference/test_convert_composer_to_hf.py:13]:
```suggestion
from composer.utils import dist, get_device
```

File: tests/a_scripts/inference/test_convert_composer_to_hf.py [hunk 4]
```diff
@@ -4,26 +4,33 @@
...
-import transformers
 from composer import Trainer
 from composer.loggers import MLFlowLogger
 from composer.utils import dist, get_device
+
+from llmfoundry.callbacks import HuggingFaceCheckpointer
+from llmfoundry.models.mpt.modeling_mpt import ComposerMPTCausalLM
+
+# Add repo root to path so we can import scripts and test it
+repo_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))
```
Comment (vchiley) [inline tests/a_scripts/inference/test_convert_composer_to_hf.py:19]:
```suggestion
repo_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), '../../../'))
```

??? or something

File: tests/a_scripts/inference/test_convert_composer_to_hf.py [hunk 5]
```diff
@@ -4,26 +4,33 @@
...
-import torch
-import transformers
 from composer import Trainer
 from composer.loggers import MLFlowLogger
 from composer.utils import dist, get_device
+
+from llmfoundry.callbacks import HuggingFaceCheckpointer
+from llmfoundry.models.mpt.modeling_mpt import ComposerMPTCausalLM
+
+# Add repo root to path so we can import scripts and test it
```
Comment (irenedea) [inline tests/a_scripts/inference/test_convert_composer_to_hf.py:18]: How come this wasn't handled by the fixtures? / Why were these tests passing in CI?

**Pull request closed without merging**